Client/Frontend/Browser/SwipeAnimator.swift

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+struct SwipeAnimatorUX {
+    var totalRotationInDegrees: Double
+    var deleteThreshold: CGFloat
+    var totalScale: CGFloat
+    var totalAlpha: CGFloat
+    var minExitVelocity: CGFloat
+    var recenterAnimationDuration: NSTimeInterval
+}
+
+private let DefaultSwipeAnimatorUX =
+    SwipeAnimatorUX(
```

Comment:
Originally no, but I have a ticket for deleting tiles off the home panel where we're going to be using the swipe animator to swipe off the tiles and the thresholds will be different there. I think renaming this to something not UX would make more sense so it doesn't get mixed up with the eventual app constants.